routers/api/v1/repo/actions_run.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2025 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package repo
+
+import (
+	actions_model "code.gitea.io/gitea/models/actions"
+	"code.gitea.io/gitea/routers/common"
+	"code.gitea.io/gitea/services/context"
+)
+
+func DownloadActionsRunJobLogs(ctx *context.APIContext) {
+	// swagger:operation GET /repos/{owner}/{repo}/actions/jobs/{job_id}/logs repository downloadActionsRunJobLogs
+	// ---
+	// summary: Downloads the logs for a workflow run redirects to blob url
```

Comment:
```suggestion
	// summary: Downloads the job logs for a workflow run
```
reasoning
- remove mention that the result is a redirect it is a http 200 download here 
  - did emulate such a redirect for artifact download just because of the weird logic in actions/download-artifact

- mention that this log is from a job not the whole workflow run